Xamarin.Forms.Core/Element.cs

```diff
@@ -359,6 +358,9 @@ protected override void OnBindingContextChanged()
 
 		protected virtual void OnChildAdded(Element child)
 		{
+			Dispatcher = DispatcherManager.Current.Dispatcher;
+			child.Dispatcher = Dispatcher;
```

Comment:
And won't the child already have a Dispatcher which was set in the constructor?

Reply:
I tested removing and had no impacts. I will make the adjustment.